tests/flow/test_function_calls.py

```
@@ -2804,3 +2804,74 @@ def test94_vector(self):
+        # outer call cannot fold either.
+        res = self.graph.query(
+            "UNWIND range(1, 200) AS i RETURN count(DISTINCT toString(randomUUID())) AS n")
+        self.env.assertEqual(res.result_set[0][0], 200)
+
+    def test97_folded_calls_keep_their_errors_and_empty_batches(self):
+        # Evaluating once must not move when an error is raised. A bad constant
+        # argument still fails the query...
+        try:
+            self.graph.query("UNWIND range(0, 9) AS i RETURN split(1, 2)")
+            self.env.assertTrue(False, depth=1)
+        except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)